package.json

```diff
@@ -43,10 +43,10 @@
     "eslint-plugin-react": "^7.24.0",
     "eslint-plugin-react-hooks": "^4.2.0",
     "history": "^5.0.0",
-    "jest": "^24.9.0",
-    "jsonfile": "^5.0.0",
+    "jest": "^26.6.3",
```

Comment:
So Jest is actually on v27 now, but that version comes with some pretty annoying changes that breaks our `react-native` tests. Gonna circle back here eventually when I update the RN dev dependency, but probably not a big priority. 

Context: https://github.com/callstack/react-native-testing-library/issues/743